### PR TITLE
payment non prod kustmization

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-app
+spec:
+  releaseName: ccpay-payment-api
+  values:
+    java:
+      environment:
+        TRUSTED_S2S_SERVICE_NAMES: "cmc,cmc_claim_store,probate_frontend,divorce_frontend,ccd_gw,bar_api,api_gw,finrem_payment_service,ccpay_bubble,jui_webapp,xui_webapp,fpl_case_service,iac,probate_backend,nfdiv_case_api,civil_service,prl_cos_api,adoption_web,refunds_api"
+        DUMMY_RESTART_VAR: false
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml:11]: is `DUMMY_RESTART_VAR `needed? default value is false from ccpay-payment-app.yaml

Comment (thomast1906) [reply k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml:11]: https://github.com/hmcts/cnp-flux-config/blob/payment-api-non-prod-v1/k8s/namespaces/fees-pay/ccpay-payment-app/ccpay-payment-app.yaml#L18 reference

File: k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-app
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml:4]: please match `releasename`

File: k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml (hunk 1 above)
Comment (AnilDhawaleGitHub) [reply k8s/namespaces/fees-pay/ccpay-payment-app/aat.yaml:11]: kept DUMMY_RESTART_VAR  in env specific file and removed from ccpay-payment-app.yam

**Review by thomast1906 — APPROVED** [review]

**Comment by AnilDhawaleGitHub** [comment]
closing it

**Pull request closed without merging**